packages/core/src/Stamp.js

```diff
@@ -31,17 +31,18 @@ const sizes = {
 }
 
 const variations = {
-  regular: css`
+  outline: css`
     color: ${getPaletteColor('base')};
     border-color: ${props => getPaletteColor(props.borderColor, 'base')(props)};
     ${props =>
       props.bg
         ? `background-color: ${getPaletteColor(props.bg, 'base')(props)};`
         : ''}
   `,
-  solid: css`
+  fill: css`
```

Comment:
Would this be a breaking change for existing usages of `Stamp`?

Reply:
Unfortunately yes, but we only added the version the other day, we could remove that release?

Reply:
I think unpublishing that release would probably be the best course of action, rather than immediately making another major. @sdalonzo, thoughts?

Reply:
could we alias it to the old name temporarily, and remove it in v4?